cloud-controller-manager/do/firewall_controller.go

```diff
@@ -97,6 +103,8 @@ type FirewallController struct {
 	serviceLister      corelisters.ServiceLister
 	fwManager          firewallManager
 	metrics            metrics
+	// nodeport services that need to be synced
```

Comment:
Not sure I understand this comment. 🤔

Reply:
oops, I think that was copy pasta'd from somewhere

Reply:
will remove that